### DISCUSSION TITLE
Formatting datetime in flow/template

### DISCUSSION THREAD

**Discussion by fa-briz-io** [directus/directus#14813] [Archive]
Hello everybody,

I made a flow that triggers a notification to a user. In the message of the notification I use {{ shift_data.start }} that translates to 2022-08-03T12:00:00. I would like to be able to format this differently. For example to: 03/08/2022 12:00. Is there a way I can reformat this? Thank you for your input!

**Comment by rijkvanzanten** [comment]
There's currently no real way to do that efficiently. I think https://github.com/directus/directus/discussions/13944 would be great for this kind of use case

**Comment by fa-briz-io** [comment]
Hoi Rijk, thank you for your reply! I'll keep a look out for a script options in the flows, that would be excellent! In case someone comes across this issue, I made a PHP script that converts the date into a format that I like and return this as JSON. In my flow I call this as a webhook. Not so elegant but working for now. Thanks again!
